Presentation.Web/Tests/Helpers/SystemCatalogHelper.ts

```diff
@@ -31,6 +35,27 @@ class SystemCatalogHelper {
             .then(() => browser.switchTo().alert().accept());
     }
 
+
+    public static deleteSystemWithoutBrowserWait(name: string) {
+        console.log(`Deleting system: ${name}`);
+        return SystemCatalogHelper.pageObject.getPage()
+            .then(() => SystemCatalogHelper.waitForKendoGrid())
+            .then(() => SystemCatalogHelper.findCatalogColumnsFor(name).first().click())
+            .then(() => browser.wait(SystemCatalogHelper.systemPage.isDeleteButtonLoaded(),SystemCatalogHelper.waitUpTo.twentySeconds))
+            .then(() => SystemCatalogHelper.systemPage.getDeleteButton().click())
+            .then(() => this.testFixture.disableAutoBrowserWaits())
```

Comment:
Hvis den fejler midt i det hele, er resten af testene i en dårlig tilstand. Sørg for at afterEach slår det til igen (browser waits). Gælder også hvis det er anvendt andre steder i dette PR

Reply:
Dette er der tjek for :) i afterEach kalder den enableAutoBrowserWaits()

Reply:
@mbkstrongminds Det her er jo en "helper", så det gør det lidt farligt med sådan en konstruktion. Er du sikker på at denne del, hører hjemme i en "helper". Handler det ikke om den konkrete test vedrørende teksten der kommer ud. Er hjælperen ikke blevet lidt for "bundet" til en konkret test? Det giver jo kun mening i den test, at browser waits slås fra. Pas på med at putte for meget viden ind i helpers, der som udgangspunkt skal realisere "generelt anvendelig" kode. Denne helper ved nu rigtig meget om en konkret test, og så skal den der, der er det nok over i selve testen. Så kan du også tale om beforeEach og afterEach. Det kan du ikke i denne kontekts.

Reply:
Ret beset kan alt indtil "disable"... beholdes og så er metoden "getSystemDeleteButton", men lige så snart det andet går igang, så er vi i testens kontekst og "the danger zone"